vtexdocs/RapiDoc
# Feature/request section

#### What is the purpose of this pull request?

To update and style the request section.

#### What problem is this solving?

Inadequate structure and styling to follow [the expected design](https://www.figma.com/file/Lx6sXdrw9KEvtSEKWttmv8/Developer-Portal?node-id=2150%3A138907) for the section showing the request params(with default values).

#### How should this be manually tested?

Check if there are any typos or styling errors in the added code. Following the new pattern to testing-version tags, you can view the achieved results on [this deploy preview](https://deploy-preview-44--elated-hoover-5c29bf.netlify.app/).

#### Screenshots or example usage
| Before | After |
|--------|-------|
| ![image](https://user-images.githubusercontent.com/42784961/172172008-79ab3479-558c-42ef-ae5b-28b0b40c2d3c.png)|![image](https://user-images.githubusercontent.com/42784961/172172192-b49d7766-0222-4bbc-b7d1-25b90162f06f.png)|
#### Types of changes
